src/Microsoft.ML.FastTree/doc.xml

```diff
@@ -102,43 +66,29 @@
         </list>
       </remarks>
     </member>
-    <example name='FastForestBinaryClassifier'>
-      <example>
-        <code language="csharp">
-          new FastForestBinaryClassifier
-          {
-            NumTrees = 100,
-            NumLeaves = 50,
-            Calibrator = new FixedPlattCalibratorCalibratorTrainer()
-          }
-        </code>
-      </example>
-    </example>
-    <example name='FastForestRegressor'>
-      <example>
-        <code language="csharp">
-          new FastForestRegressor
-          {
-            NumTrees = 100,
-            NumLeaves = 50,
-            NumThreads = 5,
-            EntropyCoefficient = 0.3
-          }
-        </code>
-      </example>
-    </example>
 
-    <member name="FastTreeTweedieRegression">
-      <summary>
-        Trains gradient boosted decision trees to fit target values using a Tweedie loss function. 
-        This learner is a generalization of Poisson, compound Poisson, and gamma regression.
-      </summary>
+    <!--  
+    The following text describes the FastForest algorithm details.
```

Comment:
>FastForest [](start = 37, length = 10)

GAMs???

Reply:
right, copy-paste error

---
In reply to: [265813553](https://github.com/dotnet/machinelearning/pull/2970#discussion_r265813553) [](ancestors = 265813553)